Gem/Code/Source/RayTracingIntersectionShaderExampleComponent.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <CommonSampleComponentBase.h>
+#include <Utils/Utils.h>
+
+namespace AtomSampleViewer
+{
+    class RayTracingIntersectionShaderExampleComponent final : public CommonSampleComponentBase
```

Comment:
Lets add a comment explaining how this sample is testing Intersection RT workload.